pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/GeoSpatialTest.java

```diff
@@ -304,11 +304,11 @@ public void testStPointFunctionWithV2(boolean useMultiStageQueryEngine)
     {
       String query =
           String.format("Select "
-                  + "ST_Point(a.st_x, a.st_y, -1), "
+                  + "ST_Point(a.st_x, a.st_y, 0), "
```

Comment:
This test is for backward compatible where we used to allow 0 and 1. We never allow -1, so that shouldn't be needed. More tests are covered in `LiteralContextText`